slack_sdk/web/internal_utils.py

```diff
@@ -229,7 +229,7 @@ def _next_cursor_is_present(data) -> bool:
 
 def _to_0_or_1_if_bool(v: Any) -> Union[Any, str]:
```

Comment:
We can modify the type hint if we change the value. `Union[Any, int]` looks uncommon, though.
```suggestion
def _to_0_or_1_if_bool(v: Any) -> Union[Any, int]:
```